### DISCUSSION TITLE
What is < new user ID > referring to at the top of the webchat panel?

### DISCUSSION THREAD

**Discussion by mareekuh** [microsoft/BotFramework-Composer#7098] [Design]
Run Composer
Open Webchat
Notice the upper left corner label says : Restart all bots - new user ID
What does new User ID refer to?
Even the dropdown lists this - and it's not actionable/cannot be changed

Expected:
Label copy in upper left corner says :
Restart < bot name >

**Comment by mewa1024** [comment]
@mareekuh, noted and will track for more feedback. The "Restart all bots - new/same user ID" split button is a carry over from Emulator, so users should know what it does.

As far as the buttons being not actionable/cannot be changed, the split button is intended to allow one click for "Restarting all bots" using as a default new user ID. You can use the down arrow to the right of the | to restart the bot with the same user ID, which also sets "same user ID" as the default going forward.

**Comment by mewa1024** [comment]
Converting this issue to a discussion, so it doesn't interfere with issue counts.
